### DISCUSSION TITLE
libmp4v2 requirement for Linux

### DISCUSSION THREAD

**Discussion by seanyar** [yermak/AudioBookConverter#250] [Q&A]
Since the pre requisite for debian version of AudioBookconverter - libmp4v2 does not seem to be maintained anymore and this package is not available in the downstream repos for bionic/focal, is there an alternative that can be used? Or is there another way to install libmp4v2

**Comment by yermak** [comment]
if .deb does not work for you - i would recommend to go for AudioBookConverter-static-binaries-x.y.z.tar.gz
It has mp4v2 statically built, there is no simple way to fix .deb for new versions of ubuntu - i believe they stopped supporting it from 20, as it was fine in 18.

**Reply by seanyar** [reply]
Thanks Yermak! That worked

**Reply by seanyar** [reply]
Static version seems to be missing 2 deps.
MP4info...Was able to install separately for Focal.
MP4art..Couldnt find a suitable installation candidate

Any thoughts/advice?

**Reply by yermak** [reply]
@seanyar, those dependencies are definitely included into archive, if they are not found, you could experiment with file called files.properties in lib/app/external.

Or try to change you working directroy to app directory before you launch the app. Most likely if your current directory /home/username and you run app with command /opt/AudioBookConverter/bin/AudioBookConverter app is searching for MP4info and MP4art relatively to you working directroy which is your home.

**Reply by seanyar** [reply]
Thanks. That helped!
